tests/monitoring/component_monitoring.go

```diff
@@ -161,34 +161,42 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 	})
 
 	Context("Errors metrics", func() {
```

Comment:
in `Fix flakiness and reenable errors metrics tests` commit - can you elaborate how this flakiness is fixed in this commit?

Reply:
Due to the destructive nature of the tests, sometimes in the cleanup phase of each test, the config had different versions, the simple step of reverting the deleted CRBs before restoring `virt-operator` of being scaled to 0, ensures the components go back to a healthy state much faster.

Reply:
sounds logical. Can you add it to the commit desc?

Reply:
added